src/components/ui/DataGrid/DataGrid.svelte

```diff
@@ -273,6 +273,7 @@ This has been seen to result in unintended and often glitchy behavior, which oft
       animateRows: false,
       columnDefs,
       doesExternalFilterPass,
+      excludeHiddenColumnsFromQuickFilter: false,
```

Comment:
This impacts other tables which may not want this behavior - could you pass this in as a prop?